pkg/workers/exec/konnector.go

```diff
@@ -79,12 +80,36 @@ type konnectorResult struct {
 
 // beforeHookKonnector skips jobs from trigger that are failing on certain
 // errors.
-func beforeHookKonnector(req *jobs.JobRequest) (bool, error) {
-	if req.Manual || req.Trigger == nil {
+func beforeHookKonnector(job *jobs.Job) (bool, error) {
+	konnectorMessage := struct {
+		Account      string `json:"account"`
+		Konnector    string `json:"konnector"`
+		FolderToSave string `json:"folder_to_save"`
+	}{}
+
+	if err := json.Unmarshal(job.Message, &konnectorMessage); err == nil {
+		inst, err := instance.Get(job.DomainName())
+		if err != nil {
+			return false, err
+		}
+		registries, _ := inst.Registries()
+		app, err := registry.GetApplication(konnectorMessage.Konnector, registries)
+		if err != nil {
+			job.Logger().Errorf("konnector %q could not get application to fetch maintenance status", konnectorMessage.Konnector)
```

Comment:
As I understand, it does not block developers to run konnectors that are not yet on the apps registry to test them. Am I right?

Reply:
Yes it was the motivation. Also, we prefer running konnectors (even if in maintenance) if the registries has issues to respond to the stack.